src/pages/events/index.svelte

```diff
@@ -1,90 +1,164 @@
 <script>
-import EventFilters from 'components/EventFilters.svelte'
-import FilteredDisplay from 'components/FilteredDisplay.svelte'
-import { isParticipant, populateEventFilterFrom } from 'data/eventFiltering.js'
-import { join, events, loading } from 'data/events.js'
-import { me } from 'data/user.js'
-
-import Icon from 'fa-svelte'
-import { faExternalLinkAlt, faEdit } from '@fortawesome/free-solid-svg-icons'
-import { goto, params } from '@roxi/routify'
-import { flip } from 'svelte/animate';
-import { fade } from 'svelte/transition';
-
-const format = date => new Date(date).toLocaleDateString([], {
-  month: 'short',
-  day: 'numeric',
-  year: 'numeric',
-  timeZone: 'UTC',
-})
-const logoUrl = event => event.image_file?.url || ''
-
-let eventFilter = {}
-
-$: eventFilter = populateEventFilterFrom($params, $me)
+  import EventFilters from "components/EventFilters.svelte";
+  import FilteredDisplay from "components/FilteredDisplay.svelte";
+  import {
+    isParticipant,
+    populateEventFilterFrom,
+  } from "data/eventFiltering.js";
+  import { join, events, loading } from "data/events.js";
+  import { me } from "data/user.js";
+
+  import Icon from "fa-svelte";
+  import { faExternalLinkAlt, faEdit } from "@fortawesome/free-solid-svg-icons";
+  import { goto, params } from "@roxi/routify";
+  import * as animateScroll from "svelte-scrollto";
+  import { flip } from "svelte/animate";
+  import { fade } from "svelte/transition";
+
+  const format = (date) =>
```

Comment:
I would prefer not using parens if they aren't needed unless we are using them consistently throughout the the code.

Reply:
Somewhere else in the code I saw someone using this. I kinda assumed, it being a `const`, that it might have some sort of better memory management instead of a regular `function`. But I can change it.

Reply:
Actually, wait, I'm pretty sure this wasn't me who did that code. It was already here. The reason there are '+'s on the side is that my editor keeps doing stuff automatically, I think to tidy the code up.

Reply:
I think @hobbitronics was just suggesting to remove the parens around `date`.

Reply:
arrow functions are just concise syntax not needing () or {} and implying return, but if you need to reference 'this' they won't work.

Reply:
```suggestion
  const format = date =>
```

Reply:
Oh, sorry, I misunderstood what you said.